R/ParseUserEnteredTable.R

```diff
@@ -168,11 +168,23 @@ parseAsVectorOrMatrix <- function(m, warn = FALSE)
     if (any(!isTextNumeric(m[-1,-1], allow.missing = TRUE)))
         row.names.given <- isQStatistic(m[1,1])
 
+    # If input data are percentages then we can do more tests
+    .is_pct <- function(x) { tmp <- asNumericVector(x); return(!is.null(attr(tmp, "statistic"))) }
```

Comment:
Should the function check that the statistic is/contains "%"?

Reply:
Currently, asNumericVector only attaches an attribute if its "%", but it is safer to check for "%". Updated in eb80adb